src/SFA.DAS.ApplyService.Application/Apply/IApplyRepository.cs

```diff
@@ -63,5 +70,11 @@ public interface IApplyRepository
         Task<IEnumerable<RoatpApplicationStatus>> GetExistingApplicationStatusByUkprn(string ukprn);
 
         Task<string> GetNextRoatpApplicationReference();
+
+        Task<bool> RecordFinancialGrade(Guid applicationId, FinancialReviewDetails financialReviewDetails, string financialReviewStatus);
+
+        Task<bool> StartAssessorReview(Guid applicationId, string reviewer);
+
+        Task<bool> StartFinancialReview(Guid applicationId, string reviewer);
```

Comment:
Can these be moved above the comment saying '// NOTE: This is old stuff or things which are not migrated over yet'

Reply:
Done